### DISCUSSION TITLE
Convert png sequence to gif failed

### DISCUSSION THREAD

**Discussion by huiwenzhang** [ImageMagick/ImageMagick#2152] [Help]
Hi, I am using ubuntu 18.04 and trying to convert a bunch of png files to gif with imagemagick convert command. However, it throws error:
```
 convert: ../../magick/quantum.c:216: DestroyQuantumInfo: Assertion `quantum_info != (QuantumInfo *) NULL' failed.
```
The version of convert is
```
Version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 http://www.imagemagick.org
Copyright: © 1999-2017 ImageMagick Studio LLC
License: http://www.imagemagick.org/script/license.php
Features: Cipher DPC Modules OpenMP 
Delegates (built-in): bzlib djvu fftw fontconfig freetype jbig jng jpeg lcms lqr ltdl lzma openexr pangocairo png tiff wmf x xml zlib
```
It said that this is because the imagemagick version is too old, so I update the imagemagick to ` 8:6.9.7.4+dfsg-16ubuntu6.8 `, the problem is I found the `convert` command still uses the 6.xx version with a soft link to a `im6.q16` file. How should I solve this?

**Comment by fmw42** [comment]
What was your exact command line? Can you post your input image?

**Reply by huiwenzhang** [reply]
Command is `convert -monitor -delay 10 -resize 640x480 recoder*.png fname.gif`. Sample images are
![recoder0000](https://user-images.githubusercontent.com/16585681/84733593-3e82b300-afd1-11ea-82e0-83cff623a30c.png)
![recoder0011](https://user-images.githubusercontent.com/16585681/84733630-58bc9100-afd1-11ea-821f-6fffe0e64ce4.png)
Images are generated by matplotlib `savefig` method which will be redrawn on each loop step.
